hw/top_earlgrey/data/ip/chip_usbdev_testplan.hjson

```diff
@@ -0,0 +1,277 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_usbdev
+  testpoints: [
+    // USBDEV integration tests
+    {
+      name: chip_sw_usbdev_vbus
+      desc: '''Verify that the USB device can detect the presence of VBUS from the USB host.
+
+            - Set up the pinmux to ensure that the SENSE/VBUS input can reach usbdev
+            - Check that VBUS is asserted indicating the presence of a connection to the
+              USB host or DPI model.
+            '''
+      features: ["USBDEV.CONN.VBUS"]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "PROD_END", "RMA"]
+      tests: ["chip_sw_usbdev_vbus"]
+      bazel: []
+    }
+    {
+      name: chip_sw_usbdev_pullup
+      desc: '''Verify that the USB device can assert the pull up to indicate its presence.
+
+            - This test extends from `chip_sw_usbdev_vbus` by checking that the DP line of
+              the USB is deasserted after VBUS detection.
+            - Enable the interface by asserting the pull up.
+            - Check that the DP line rises by reading from the `phy_pins_sense` register.
+            - Host/DPI model detects the presence of a Full Speed device.
+            - Deassert the pull up.
+            - Check that the DP line falls again.
+            - For those targets which can apply a pull up to either line, repeat the above
+              for the DN line using the usbdev pin-flipping feature.
+            - No other communication with the host, so indicating Low Speed is harmless.
```

Comment:
Indeed, some of these newly-created simple tests are a bit 'rude' but they're intended to provide a series of stepping stones to test USB connectivity and signaling incrementally, since `usbdev_test` is pretty much 'all or nothing' works.